packages/components/organisms/f-checkout/test/component/f-checkout-delivery.component.desktop.spec.js

```diff
@@ -5,30 +5,60 @@ const Checkout = require('../../test-utils/component-objects/f-checkout.componen
 let checkout;
 
 describe('f-checkout "delivery" component tests', () => {
-    beforeEach(() => {
-        checkout = new Checkout('organism', 'checkout-component');
-        checkout.withQuery('&knob-Service Type', 'delivery')
-            .withQuery('&knob-Is User Logged In', true)
-            .withQuery('&knob-Is ASAP available', true);
+    describe('uk tenant', () => {
+        beforeEach(() => {
+            checkout = new Checkout('organism', 'checkout-component');
+            checkout.withQuery('&knob-Service Type', 'delivery')
+                .withQuery('&knob-Is User Logged In', true)
+                .withQuery('&knob-Is ASAP available', true);
 
-        const pageUrl = buildUrl(checkout.componentType, checkout.componentName, checkout.path);
+            const pageUrl = buildUrl(checkout.componentType, checkout.componentName, checkout.path);
 
-        checkout.open(pageUrl);
-        checkout.waitForComponent();
+            checkout.open(pageUrl);
+            checkout.waitForComponent();
+        });
+
+        it('should enable a user to submit a postcode with correct characters', () => {
+            // Arrange
+            const addressInfo = {
+                postcode: 'AR51 1AA'
+            };
+
+            // Act
+            checkout.clearBlurField('addressPostcode');
+            checkout.populateCheckoutForm(addressInfo);
+            checkout.goToPayment();
+
+            // Assert
+            expect(checkout.isPostcodeTypeErrorDisplayed()).toBe(false);
+        });
     });
 
-    it('should enable a user to submit a postcode with correct characters', () => {
-        // Arrange
-        const addressInfo = {
-            postcode: 'AR51 1AA'
-        };
+    describe('au tenant', () => {
```

Comment:
Do we want to support `nz` too?

Reply:
I will have to double check but as mentioned above, AU is the only country that requires the administrative Area field on the API